server/server_account_test.go

```diff
@@ -431,6 +431,41 @@ func TestAccount_Delete_Not_Allowed(t *testing.T) {
 	require.Equal(t, 40026, toHTTPError(t, rr.Body.String()).Code)
 }
 
+func TestAccount_Delete_Success_WithWebPush(t *testing.T) {
```

Comment:
There's already `TestServer_WebPush_DeleteAccountUnsubscribe` in the `sever_webpush_test.go`, may have put it in the wrong place haha

Reply:
I'll remove the other one